bool cmd
bool topic
string  path
string num_name
---
int32 result